functional-tests/README.md

```diff
@@ -0,0 +1,7 @@
+
+<!--VM options = -Dche.threads=1 -Dche.workspace_pool_size=1 -Dche.host="rhche.prod-preview.openshift.io" -Dche.port=443
```

Comment:
this does not look like a good README for me. Could you please add some wording not just comments from `pom.xml`

Reply:
ok, cool that this is on a track. It is ok for me if README would be updated in the another PR